pages/services/data-science-engine/1.0.0/install/index.md

```diff
@@ -0,0 +1,402 @@
+|----------------|----------------|
+| Disabled | Not available |
+| Permissive | Optional |
+| Strict | Required |
+
+For more information about service accounts, see the [Security](/services/data-science-engine/1.0.0/security/) documentation. 
+
+# Default installation
+
+## From the CLI
+
+To install the {{ model.techName }} service from the DC/OS CLI, run the following command:
+
+```bash
+dcos package install  {{ model.packageName }}
+```
+
+This installs the {{ model.packageName }} service, {{ model.packageName }} CLI, dispatcher, and, optionally, the history server. 
+
+<a name="CLI"></a>
+
+## Installing the {{ model.packageName }} CLI
+If for some reason you did not install the {{ model.packageName }} CLI, you can do so with this command. This is useful if you already have a {{ model.packageName }} cluster running, but need the {{ model.packageName }} CLI.
+
+```bash
+dcos package install {{ model.packageName }} --cli
+```
+
+### View deployment
+
+Go to the **Services >Deployments** tab of the DC/OS UI to monitor the deployment. When it has finished deploying, visit {{ model.packageName }} at `http://<dcos-url>/service/ {{ model.packageName }}/`.
+
+<a name="ui"></a>
+
+## From the UI
+
+You can install {{ model.techName }} from the DC/OS Catalog. The interface provides either a graphical view of the configuration screens, or a split-window CLI interface which lets you edit or create a JSON file. The default values are pre-loaded for each field; you can change these default values as you need using either side of the split window.
+
+If you install {{ model.techName }} via the DC/OS GUI, you must install the {{ model.packageName }} CLI as a separate step (see <a name="CLI">Installing the {{ model.packageName }} CLI</a>.
+
+
+1. From the Catalog, select {{ model.techName }}.
+
+![Catalog view]()
+Figure 1 - Catalog showing {{ model.packageName }} package
+
+1. Review the **Preinstall Notes** to make sure you understand all the options available at install time. Click **Review & Run** when you are finished.
+
+![Preinstall notes]()
+Figure 2 - Preinstall Notes
+
+## Configuration screens
+
+There are seven configuration tabs in the **Edit Configuration** page. The fields will either be blank or will be filled in with the default values. If you prefer to configure your service using a JSON file, click on the **JSON Editor** button in the upper right to open a "split-screen" view. Any changes made to either side of this view will be reflected in the other.
+
+### Service 
+The first configuration tab you will edit is the Service tab. In these fields, you will configure the service name, define the CPUs allocated to it, and set other fields. 
+
+### OIDC 
+The second configuration tab is the OIDC tab, where  you will configure client IDs, secrets, and other identification values.
+
+### S3
+
+The third configuration tab is the S3 tab, where you can define the AWS region, endpoint, and any other values required.
```

Comment:
You're going to add screenshots for these sections?

Reply:
As soon as I get any. :)